sklearn/decomposition/sparse_pca.py

```diff
@@ -312,6 +380,10 @@ class MiniBatchSparsePCA(SparsePCA):
         Per-feature empirical mean, estimated from the training set.
         Equal to ``X.mean(axis=0)``.
 
+    explained_variance_ : array, [n_components]
+        The explained variance versus component.
```

Comment:
`array, shape (n_components,)`

Reply:
also it might be a good idea to test that `explained_variance` is the same whether `normalize_components` is True or False.